source/NVDAObjects/UIA/__init__.py

```diff
@@ -1186,13 +1186,15 @@ def findOverlayClasses(self,clsList):
 		if self.windowClassName == "ConsoleWindowClass":
 			from . import winConsoleUIA
 			winConsoleUIA.findExtraOverlayClasses(self, clsList)
-		elif UIAClassName in ("TermControl", "TermControl2"):
+		elif UIAClassName in ("TermControl", "TermControl2", "WPFTermControl"):

```

Comment:
It might be worth moving this tuple out to a named frozen set, with the relevant comments attached

Reply:
In this PR, or a separate one?

Reply:
in this PR, e.g. `_TERMINAL_CLASS_NAMES = frozenset(...)`